distribution/bin/check-licenses.py

```diff
@@ -225,6 +225,9 @@ def build_compatible_license_names():
     compatible_licenses['Apache License v2.0'] = 'Apache License version 2.0'
     compatible_licenses['Apache License, version 2.0'] = 'Apache License version 2.0'
     compatible_licenses['Apache 2.0 License'] = 'Apache License version 2.0'
+    compatible_licenses['Apache License, 2.0'] = 'Apache License version 2.0'
+    compatible_licenses['Confluent Community License'] = 'Confluent Community License'
```

Comment:
hmm, what requires the confluent community license? I don't see anything in licenses.yaml mentioning it. Last i knew, the clients to confluent things were still apache licensed (i'm not sure we can distribute anything with this license in Druid packages, would have to check into that, but I suspect not since it has field of use restrictions)

Reply:
<img width="1422" alt="Screen Shot 2021-02-07 at 10 48 58 AM" src="https://user-images.githubusercontent.com/24642075/107135019-34cc0800-6932-11eb-984d-acdf4be844a0.png">
This is dependency report generated by druid. I think it means kafka-protobuf-provider is licensed by Confluent Community License and Apache License. I change this python script because it can not work if this map lose key 'Confluent Community License'. 
<img width="1166" alt="Screen Shot 2021-02-07 at 10 56 51 AM" src="https://user-images.githubusercontent.com/24642075/107135162-4366ef00-6933-11eb-87c2-a854a57f5630.png">
<img width="1137" alt="Screen Shot 2021-02-07 at 10 57 03 AM" src="https://user-images.githubusercontent.com/24642075/107135165-46fa7600-6933-11eb-8e22-c403ef3edcb5.png">
In MVN Repository, it marked as Apache license. So can I use it in Druid packages? I'm not very sure about that.